app/jobs/build_initiated_by_job.rb

```diff
@@ -0,0 +1,24 @@
+require 'job_base'
+require 'git_repo'
+require 'partitioner'
```

Comment:
The Partitioner class is not used in this job

Reply:
:/ copy pasta